README.md

```diff
@@ -5,6 +5,27 @@ A Slicer extension that enables user to interact with a Slicer scene using virtu
 
 ![](VR.png)
 
+Building steps
+--------------
+
+- Clone Slicer from https://github.com/jbvimort/Slicer/tree/VR and checkout [VR](https://github.com/jbvimort/Slicer/tree/VR) branch
```

Comment:
https://github.com/jbvimort/Slicer/tree/VR -> https://github.com/jbvimort/Slicer

Reply:
and checkout  -> and checkout  the